subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/configuration/InstrumentationPluginConfiguration.kt

```diff
@@ -32,6 +32,8 @@ public object InstrumentationPluginConfiguration {
          */
         public var useService: Boolean = false
 
+        public var useInMemoryReport: Boolean = false
```

Comment:
Is it a temporary feature toggle or a long term config?
How about enum? 

report = [in-memory, avito-report]

Reply:
temporary
report config already has it's own `testReport` function, which i don't want to interfere

Reply:
Ok. We could express it explicitly in API:

```kotlin
experimental {
   report.inMemory = true
}
```

Reply:
fixed, take a look

Reply:
Thanks, looks self-explanatory